Bug 1183160 - [Bugzilla Lite] Testing the Log out to Bugzilla Lite

mermi opened the pull request:
Need review. Thanks

funktr0n committed:
Bug 1142104 - Progress bar now announces loading/loaded

mermi committed:
Testing log in bugzilla lite

mermi committed:
change what is noted

mermi committed:
change some incorrect things from the last commit

mermi committed:
app file need syntax review

mermi committed:
remove no need files

mermi committed:
made correction from the last notes

mermi committed:
problem wth network connection

mermi committed:
fix network connection problem

mermi committed:
remove typo

mermi committed:
test with the restart option

mermi committed:
test for logout

autolander commented:
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=2527ca0f2ac937978f32ef8c758d59d26702f963

mermi committed:
add assert for the logout

autolander commented:
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6ba26b39942c097b84d053aa09506e33def94ee3

mancas committed:
Bug 1178418 - When importing a multiple contacts vcard, if a contact does not have Company field, "false" is shown in the import contacts list

lissyx committed:
Bug 1184476 - Handle NotFoundError from DOMError r=daleharvey

gabrielesvelto committed:
Bug 1147237 - Do not wait for the mozL10n.ready event and clean up the tests so that they are fully stateless r=kgrandon

bzbarsky committed:
Bug 1184802.  Remove vestigial uses of mozRequestAnimationFrame from gaia code

Commit:
Bug 1184820 - update imagecompare tests to accomodate changes in Bug 1180819
add options to take_screenshot() so user can customize the prewait
Increase the default fuzz factor to 15%
Adding extra wait after zoom in action

gabrielesvelto committed:
Merge pull request #31005 from gabrielesvelto/bug-1147237-do-not-listen-to-mozl10n-ready

Bug 1147237 - Do not wait for the mozL10n.ready event and clean up the tests so that they are fully stateless r=kgrandon

lissyx committed:
Merge pull request #31004 from lissyx/bug1184476

Bug 1184476 - Handle NotFoundError from DOMError r=daleharvey

Commit:
Bug 1184981 - Gaia UI test failures because open_wifi_settings and open_cell_and_data_settings don't exist anymore

KevinGrandon committed:
Bug 1180068 - Disable intermittent apps/search/test/marionette/places_search_test.js | Places tests Test url searching

Commit:
Bug 1165765 - Bump chunk count to 40 for Gij. r=me a=testonly

nullaus committed:
Merge pull request #31029 from nullaus/master

Bug 1165765 - Bump chunk count to 40 for Gij. r=me a=testonly

Commit:
Merge pull request #31026 from npark-mozilla/1184981

Bug 1184981 - Gaia UI test failures because open_wifi_settings and open_cell_and_data_settings don't exist anymore

Commit:
Merge pull request #31014 from npark-mozilla/update

Bug 1184820 - update imagecompare tests to accomodate changes in Bug 1180819

KevinGrandon committed:
Bug 1180534 - Follow-up, fix developer tool HUD colors

timdream committed:
Bug 1184802 - Part 0, Stop compare sizes of each layout in keyboard_test.js

timdream committed:
Merge pull request #31034 from timdream/keyboard-build-test-remove-size-check2

Bug 1184802 - Remove vestigial uses of mozRequestAnimationFrame from gaia code, r=timdream

rvandermeulen committed:
Merge pull request #30560 from FunkTron/Bug1142104

Bug 1142104 - Progress bar now announces loading/loaded

gasolin committed:
Bug 1159491 - [RTL] [Settings] - The linking arrow does not align left in the Built-in Keyboard menu, r=tim

gasolin committed:
Merge pull request #31036 from gasolin/issue-1159491

Bug 1159491 - [RTL] [Settings] - The linking arrow does not align left in the Built-in Keyboard menu, r=tim

jpruden92 committed:
Merge pull request #30890 from mancas/bug1181996

Bug 1181996 - Create basic 'matching' view in a separate .html

borjasalguero committed:
Merge pull request #30978 from mancas/bug1178418

Bug 1178418 - When importing a multiple contacts vcard, if a contact …

mermi committed:
change class name

autolander commented:
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=cc0c9574a2e2693140dd7de8ff9b23abc329dacd

autolander commented:
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4e1a6d57f1bbc3211fe1d5b86241fecbf0c08ae8

autolander commented:
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4e1a6d57f1bbc3211fe1d5b86241fecbf0c08ae8
